yb-voyager/src/migassessment/sizing_test.go

```diff
@@ -459,8 +459,8 @@ func TestCheckShardedTableLimit_WithinLimit(t *testing.T) {
 	recommendation := map[int]IntermediateRecommendation{
 		16: {
 			ShardedTables: []SourceDBMetadata{
-				{SchemaName: "public", ObjectName: "table1", Size: 10.0, ReadsPerSec: 100, WritesPerSec: 50},
-				{SchemaName: "public", ObjectName: "table2", Size: 10.0, ReadsPerSec: 100, WritesPerSec: 50},
+				{SchemaName: "public", ObjectName: "table1", Size: 10.0, ReadsPerSec: sql.NullInt64{Valid: true, Int64: 100}, WritesPerSec: sql.NullInt64{Valid: true, Int64: 50}},
+				{SchemaName: "public", ObjectName: "table2", Size: 10.0, ReadsPerSec: sql.NullInt64{Valid: true, Int64: 100}, WritesPerSec: sql.NullInt64{Valid: true, Int64: 50}},
```

Comment:
I see this repeated multiple times: 
1. `sql.NullInt64{Valid: true, Int64: 100}`
2. `sql.NullInt64{Valid: true, Int64: 50}}`

Maybe a global const variable or something and reusing it at all places will be helpful?

Reply:
I agree that some values are reused in test case and we can create a constant. But, what I've learned over the years is that the test data should be isolated and self explanatory. Also, there are tests which are using different values. Hence, decided to add multiple occurrences. I'm also thinking of creating a separate fixtures/data file for this test data, this is something I plan to do in next cycle.